### PR TITLE
systemd

### PR DESCRIPTION
**Pull request by jburel**
Remove env variable and fix path
This should remove the 502 bad gateway 
reported here https://trello.com/c/jYST0HL6/133-omeroweb-install-virtualenv-systemd-service-file-doesnt-work

cc @manics

### CODE REVIEW

**Comment by jburel** [comment]
Updated as suggested here https://www.openmicroscopy.org/community/viewtopic.php?f=5&t=8346&start=10

File: ansible/templates/centos7/omero-web-systemd.service.j2 [hunk 1]
```diff
@@ -14,9 +14,9 @@ Type=forking
 PIDFile={{ omero_user_home_dir }}/OMERO.py/var/django.pid
 Restart=no
 RestartSec=10
-Environment="PATH={{ virtualenv_path }}/bin:/bin:/usr/bin" "WEBBINDIR={{ omero_user_home_dir }}/OMERO.py/bin"
-ExecStart={{ virtualenv_path }}/python $WEBBINDIR/omero web start
-ExecStop={{ virtualenv_path }}/python $WEBBINDIR/omero web stop
+Environment="PATH={{ virtualenv_path }}/bin:/bin:/usr/bin"
+ExecStart={{ virtualenv_path }}/bin/python /home/omero/OMERO.py/bin/omero web start
```
Comment (manics) [inline ansible/templates/centos7/omero-web-systemd.service.j2]: I think `/home/omero` should be `{{ omero_user_home_dir }}`

Comment (jburel) [reply ansible/templates/centos7/omero-web-systemd.service.j2]: forgot to re-establish that one
thanks for spotting it

**Comment by jburel** [comment]
@manics done

**Comment by jburel** [comment]
https://github.com/ome/omeroweb-install/pull/24 fixes the false positive
The change here fixes the issue for centos7 see https://travis-ci.org/ome/omeroweb-install/builds/268314100 (build combining this PR and gh-24)

**Pull request merged**